### ISSUE TITLE
Is declined assets report missing?

### ISSUE THREAD

**Issue by JohnBlunden** [grokability/snipe-it#7658]
**Server (please complete the following information):**
 - Snipe-IT Version: v4.6.15 build 4011 (gd1e9fbfa2)
 - OS: Windows Server 2012 R2
 - Web Server: IIS
 - PHP Version: 7.2.14

**Is your feature request related to a problem? Please describe.**
Yes, when we check out assets to users, they get an email where they have to accept and sign for it. until they have accepted, the assets can be listen under Reports--> unaccepted assets. That is fine.
But if the user declines the asset, there does not seem to be any report, not even under custom reports, that can list those. Is there a way of listing those, or is that feature missing? We need to be able to keep track of all the declined ones.

**Describe the solution you'd like**
Add a report option for declined assets.

**Describe alternatives you've considered**
At this point we have no other way to track this than to write down all the checked out assets and then manuelly go and check history of those assets to see if the have been declined which takes a lot of time.

**Additional context**

**Comment by JohnBlunden** [comment]
Yes it is still relevant, unless recent version has added this feature? I have not updated SnipeIT for 8 months or so.

**Comment by sanjurofla** [comment]
Very relevant, the report is needed. It would be very beneficial to have the email sent out when user declines/accepts an asset.

**Comment by SirajQu** [comment]
This is very important feature as it's difficult to go each asset and check for the declined assets.

**Comment by etodd3** [comment]
still relevant

**Comment by JohnBlunden** [comment]
Still relevant

**Comment by sanjurofla** [comment]
Relevant

Sent from phone.

________________________________

**Comment by SirajQu** [comment]
Yes still

**Comment by Speed1** [comment]
This whould be nice +1

**Comment by morgish** [comment]
Very relevant.

**Comment by ChristophAuch** [comment]
For us this feauture is also relevant, in addition it would be useful to send a mail to the processor when the asset he handed out was declined.

**Comment by TomFabiano** [comment]
This is still relevant. Decline from user should results in email notification to issuer + separate tab listing declines.

**Comment by kazzyt** [comment]
We are interested in this also.  I would have thought if someone declines an asset the person who provided it would be notified.

**Comment by triple-HA** [comment]
I'm also interested with that notification

**Comment by golebiewsky** [comment]
We are also interested! Declined assets just go back to being unassigned, but what if the assets were already shipped to the user..?

**Comment by JeffBlai** [comment]
This is needed.  We currently have to manually keep a spreadsheet of this info.

**Comment by eddy445** [comment]
Declined assets go back to being ready to deploy, it would be nice to have an alert when this happens.

**Comment by swift2512** [comment]
This came a long way...

**Comment by swift2512** [comment]
It's possible to filter Activity Report to only contain Declined entries.
